draft-ietf-httpbis-client-hints.md

```diff
@@ -109,7 +109,7 @@ Servers can advertise support for Client Hints using the Accept-CH header field
 Accept-CH is a Structured Header {{!I-D.ietf-httpbis-header-structure}}. Its value MUST be a list-of-tokens (Section 3.3 of {{!I-D.ietf-httpbis-header-structure}}) whose members are tokens (Section 3.9 of {{!I-D.ietf-httpbis-header-structure}}). Its ABNF is:
 
 ~~~ abnf7230
-  Accept-CH = #sh-list-of-tokens
+  Accept-CH = sh-list-of-tokens
```

Comment:
`sh-list-of-tokens` hasn't been defined as ABNF; you probably need to use just `sh-list` and say its contents are tokens in prose.

Reply:
Thanks, fixed it!